strictEqual.js

```diff
@@ -0,0 +1,10 @@
+const strictEqual = (valueOne, valueTwo) => {
```

Comment:
Make sure you turn on the `Format on Save` setting in VS Code, I'm noticing some spacing things that VS Code will clean up for you that will make things neater to read.